src/saltext/vmware/states/esxi.py

```diff
@@ -957,13 +965,105 @@ def advanced_config(
     return ret
 
 
+def advanced_configs(
+    configs,
+    name="advanced_configs",
+    datacenter_name=None,
+    cluster_name=None,
+    host_name=None,
+    service_instance=None,
+    profile=None,
+    less=False,
+):
+    """
+    Set advanced configuration on matching ESXi hosts.
+
+    configs
+        Set of key value pairs to be set on matching ESXi hosts (required)
+
+    datacenter_name
+        Filter by this datacenter name (required when cluster is specified)
+
+    cluster_name
+        Filter by this cluster name (optional)
+
+    host_name
+        Filter by this ESXi hostname (optional)
+
+    service_instance
+        Use this vCenter service connection instance instead of creating a new one. (optional).
+
+    profile
+        Profile to use (optional)
+
+    .. code-block:: yaml
+
+        Remove User:
+          vmware_esxi.advanced_configs:
+            - name: Annotations.WelcomeMessage
+            - value: Hello
```

Comment:
**suggestion** this example looks like it was copy-pasted from the other state, and doesn't look like it will actually work with this state.

Reply:
Changed to match the actual sls